### PR TITLE
RockPaperScissors

### PR DESCRIPTION
**Pull request by Niiola**

### CODE REVIEW

File: try.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import random
```
Comment (spignelon) [inline try.py]: Unnecessary file try.py

File: main.py [hunk 2]
```diff
@@ -129,44 +154,10 @@ def respond(voice_data):
             mixer.setvolume(percentage)
 
 
-def PaperScissors(voice_data):
```
Comment (spignelon) [inline main.py]: I think here you have deleted the changed you made in your previous commit 2f817641635afd3604a5b6a750d674d5049076d4 in which you have added PaperScissors() function, an implementation of rock, paper, scissors game.

**Comment by Niiola** [comment]
I will implement it now

**Comment by Niiola** [comment]
Should I add both task at the same time and the push?

**Comment by spignelon** [comment]
@Niiola the best thing you can do is create two new pull requests, one in which you implemented the rock paper scissors game 2f817641635afd3604a5b6a750d674d5049076d4, and the other in which you have added some basic commands 16598f30a1faabe4ec4e4b33c72c4d2cb6a9eafd.

This way it will be neater, and you'll get your two separate PRs done.

**Issues #14, #18**

**Comment by Niiola** [comment]
I am unable to make 2 pull request

**Comment by spignelon** [comment]
> I am unable to make 2 pull request

https://www.youtube.com/watch?v=rgbCcBNZcdQ
Maybe search on YouTube or better check the the digital resources for hacktoberfest.

https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github

You just have to repeat it twice, once with rock paper scissors #18  and once with basic commands #14.

**Comment by Niiola** [comment]
Done

**Pull request closed without merging**